tests/unit-tests/UnitTests/Validators/IntegrityChecks/Form/BookingFormCheckTests.cs

```diff
@@ -270,7 +334,7 @@ public void BookingFormCheck_Should_Return_Failure_Message_If_FormSchema_Contain
             // Assert
             Assert.False(result.IsValid);
             Assert.Single(result.Messages);
-            Assert.Equal($"{IntegrityChecksConstants.FAILURE}Booking Element Check, Form contains different booking provider. Only one provider allows on for form", result.Messages.First());
+            Assert.StartsWith(IntegrityChecksConstants.FAILURE, result.Messages.First());
```

Comment:
can this be changed back and check the full error message rather then it just starts with FAILURE:

Reply:
It can be, but I question why that would be better. How about just checking that there is an error message then? Then it's not at all specific as to the content of a string... more generic. That sounds better to me. Any thoughts on that, or explanation as to why it should be reverted is a better unit test? All the other unit tests in this class just use the "Failure" StartsWith/DoesNotContain. One test doesn't even check the message at all.

Reply:
They mostly got changed in the refactor i think to just checking failure but for me you could eaily break this test and not be aware as its only checking Failure and the method itself returns a failure message multiple times.

Reply:
Added that Single check for Failure messages, and that it contains the bulk of the specific error's corresponding message from BookingConstants.cs